multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -64,7 +93,26 @@ func (c *Cluster) stackName() string {
 	return c.cfg.ClusterName
 }
 
-func (c *Cluster) Create(tlsConfig *TLSConfig) error {
+func (c *Cluster) initAssets(assetDir string) *TLSConfig {
+
+	c.cfg.InstallWorkerScript = blobutil.MustReadCompressFile(filepath.Join(assetDir, "scripts", "install-worker.sh"))
+	c.cfg.InstallControllerScript = blobutil.MustReadCompressFile(filepath.Join(assetDir, "scripts", "install-controller.sh"))
+
+	manifestPath := filepath.Join(assetDir, "manifests")
+	c.cfg.ClusterManifestsTar = blobutil.MustTarCompressDirectory(assetDir, filepath.Join(manifestPath, "cluster"))
+	c.cfg.ControllerManifestsTar = blobutil.MustTarCompressDirectory(assetDir, filepath.Join(manifestPath, "controller"))
+	c.cfg.WorkerManifestsTar = blobutil.MustTarCompressDirectory(assetDir, filepath.Join(manifestPath, "worker"))
+
+	credentialsDir := filepath.Join(assetDir, "credentials")
+	tlsConfig := NewTLSConfig(credentialsDir)
+	tlsConfig.ReadFilesFromPaths()
+	return tlsConfig
+}
+
+func (c *Cluster) Create(assetDir string) error {
+
+	tlsConfig := c.initAssets(assetDir)
```

Comment:
nit: This method seems a little overloaded: read/compress a bunch of different paths then return a random tls struct.

I won't block on this, but maybe we can come up with some more linear workflow. Right now it's somewhat scattered in responsibility.

If "up" expects that all information required is available in ./artifacts/$cluster then this could be as simple as:

```
cluster, err := NewCluster(assetDir)
if err return {err}
cluster.Create()
```

As it stands, "up" requires a clusterConfig which is use to generate an AWS config, then both are used to create a cluster object, which then creates a TLSconfig as part of Create() ...